reference/release-notes/1.21.0.md

```diff
@@ -0,0 +1,67 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.21.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.21.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+
+* Initial support for installing Anbox Cloud Appliance on Ubuntu 24.04.<!--AC-2229-->
+* GL Async swap support is disabled by default. See [GL Async swap support](https://discourse.ubuntu.com/t/ams-configuration/20872#gl-async-swap-support-11) for more information.<!--AC-2228-->
+* Production support for Vulkan on NVIDIA GPUs is available:
+  - The default graphics driver stack used is Vulkan, VirGL/Venus and ANGLE. Latency and flickering is improved for VirGL by introducing a dedicated component, frame renderer. <!--AC-2152-->
+  - System tracing is enabled for Android 12 and 13. This helps in improving the performance of your application. See [System tracing overview](https://developer.android.com/topic/performance/tracing) for more information. <!--AC-2161-->
+
+* In the [AMS HTTP API](https://anbox-cloud.github.io/latest/ams/), `/1.0` endpoint exposes the cluster ID to enable identifying a subcluster.<!--AC-2148-->
+
+### Streaming improvements
+
+< List enhancements to streaming stack here. >
+
+### Dashboard improvements
+
+The following improvements are available in the Anbox Cloud dashboard for this release:
+
+* Custom instance creation
+* Support multiple subclusters from a single dashboard for large scale deployments
+* Experimental "upscaling" parameter in session view
+* Experimental support for managing the Vehicle Hardware Abstraction Layer (VHAL) properties
+* Enhancements for better user experience
+* Enhancements for improved responsiveness, error handling and messages
+* Updated Python and Javascript dependencies.
+
+### Other
+
+* Android Automotive (AAOS) images with experimental VHAL support are available. See [Provided images](https://anbox-cloud.io/docs/reference/provided-images) for more details.
```

Comment:
@ajanon Once documentation for VHAL is published, I will add those links as cross-references here. But this may have to happen on the release day as I assume there will be new topics for VHAL documentation.